### ISSUE TITLE
Allow custom logic in `data` method (no return value)

### ISSUE THREAD

**Issue by joncursi** [cleverbeagle/seeder#5]
I want to add a bunch of dependent data after users have been inserted. Right now the extra `data` method requires you to return a function that specifies a single `collection` name. But this doesn't work for complex systems that have dependent data that needs to go into multiple collections, simultaneously (i.e. if you follow someone, an insert needs to happen on both the `Followers` and `Followees` collections).

It would be great if `data` did not require a return value, which would allow the developer to write any old javascript function containing their custom logic. I.e.

```
      data(userId) {
        createUserFollowships(userId);
      },
```

Notice the omission of `return`. The above code works functionally, however the Meteor console gets spammed with errors:

```
 TypeError: Cannot read property 'collection' of undefined
W20170718-15:17:47.814(-4)? (STDERR)     at Seeder.seedDependent (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/@cleverbeagle/seeder/index.js:109:50)
W20170718-15:17:47.814(-4)? (STDERR)     at Seeder.createUser (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/@cleverbeagle/seeder/index.js:96:37)
W20170718-15:17:47.814(-4)? (STDERR)     at Seeder.sow (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/@cleverbeagle/seeder/index.js:64:20)
W20170718-15:17:47.814(-4)? (STDERR)     at Seeder.seed (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/@cleverbeagle/seeder/index.js:45:23)
W20170718-15:17:47.814(-4)? (STDERR)     at new Seeder (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/@cleverbeagle/seeder/index.js:26:12)
W20170718-15:17:47.815(-4)? (STDERR)     at exports.default (/Users/jcursi/Sites/joncursi/redbird-web/node_modules/@cleverbeagle/seeder/index.js:117:10)
W20170718-15:17:47.815(-4)? (STDERR)     at users.js (imports/startup/server/fixtures/users.js:18:7)
W20170718-15:17:47.815(-4)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:333:9)
W20170718-15:17:47.815(-4)? (STDERR)     at require (packages/modules-runtime.js:228:16)
W20170718-15:17:47.815(-4)? (STDERR)     at index.js (imports/startup/server/fixtures/index.js:3:1)
```

**Comment by cleverbeagle** [comment]
@joncursi apologies, I had notifications turned off on this repo! Just came across this on a project. Will take a look.

**Comment by cleverbeagle** [comment]
Fixed and available in 1.3.0 :)

**Comment by joncursi** [comment]
Just tested it out - works great! Thank you!
